### DISCUSSION TITLE
fallback to another scaler?

### DISCUSSION THREAD

**Discussion by nahum-litvin-hs** [kedacore/keda#2660] [Ideas]
Hi all, I love KEDA.
Currently, I use mainly the **Prometheus scaler** to scale **over rabbitmq** using a complex query.
What id like to do if possible is **in case prometheus is down**. 
to autoscale using the rabbitmq scaler instead of constant values

**Comment by tomkerkhove** [comment]
That's a good idea, we don't have this on our radar yet but a valid feature request imo

**Comment by tomkerkhove** [comment]
Also, we are always happy to list end-users on keda.sh!

In case you are interested, 👉 https://github.com/kedacore/keda-docs#become-a-listed-keda-user
